src/main/resources/META-INF/plugin.xml

```diff
@@ -211,6 +211,21 @@
             <!-- In $default keymap `control` key (Windows/Linux) corresponds to `cmd` key on macOS -->
             <keyboard-shortcut keymap="$default" first-keystroke="control alt shift M" />
         </action>
+
+        <action id="GitMachete.CheckoutNextAction"
+                class="com.virtuslab.gitmachete.frontend.actions.navigation.CheckoutNextAction">
+            <keyboard-shortcut keymap="$default" first-keystroke="alt RIGHT" />
```

Comment:
Works for me on Linux! Albeit I'd use down/up for prev/next, and left for parent

Reply:
Right can also be for next, same as down

Reply:
Now because next in the layout is down, and prev is up, I set them as such to be intuitive.

Reply:
Ouch, right, I meant _down/up for **next/prev**_, not _prev/next_